packages/jest-cli/src/init/__tests__/init.test.ts

```diff
@@ -19,13 +19,16 @@ jest.mock(
   '../../../../jest-config/build/getCacheDirectory',
```

Comment:
Perhaps `getCacheDirectory` should be exposed in `jets-config`?

Reply:
is it actually useful? i.e. what use cases are there you think?

Reply:
Just to avoid reaching into `build`. Nothing else. Can stay as it is.